index.html

```diff
@@ -4258,10 +4258,27 @@ <h1>DID Methods</h1>
             <a href="https://workday.github.io/work-did-method-spec/">Workday DID Method</a>
           </td>
         </tr>
+        <tr>
+          <td>
+            did:polygon:
+          </td>
+          <td>
+            PROVISIONAL
+          </td>
+          <td>
+            Polygon (Previously MATIC)
+          </td>
+          <td>
+            Ayanworks, MATIC
+          </td>
+          <td>
+            <a href="https://github.com/ayanworks/polygon-did-method-spec">Polygon DID Method</a>
+          </td>
+        </tr>
       </tbody>
     </table>
 
   </section>
 
 </body>
-</html>
+</html>
```

Comment:
Don't change this.

Reply:
Hi, actually this line change was induced because of addition of code. We haven't altered anything on this line.